### DISCUSSION TITLE
How Copilot usage be calculated by REST API of copilot for organization

### DISCUSSION THREAD

**Discussion by shaohanlin** [community/community#123714] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I see there's REST API to get copilot usage of organization. [Get a summary of Copilot usage for organization members](https://docs.github.com/en/rest/copilot/copilot-usage?apiVersion=2022-11-28#get-a-summary-of-copilot-usage-for-organization-members)
If there are some members of my org has copilot seat which is assiged by other org. 
Will the usage of them be calculated when using API to get usage of my org?

**Comment by Akash1134** [comment, marked as answer]
Hi @shaohanlin ,

When you use the API to get Copilot usage for your organization, it will include the usage information for all members of your organization. However, if a member of your organization is using a Copilot seat assigned by a different organization, their usage will be tracked under that organization, not yours.

So, to answer your question: if a member of your organization has a Copilot seat assigned by a different organization, their usage will not be included when you fetch the usage details for your organization. Their usage will be calculated under the organization that assigned them the Copilot seat.

Thanks!

**Reply by sandeep-tz** [reply]
@Akash1134 Will the APIs be able to give us the result if repository is hosted on Bitbucket instead of Github?
